src/plone/restapi/deserializer/atcontent.py

```diff
@@ -67,6 +67,7 @@ def __call__(self, validate_all=False, data=None):
                 notify(ObjectInitializedEvent(obj))
                 obj.at_post_create_script()
             else:
+                obj.reindexObject()
```

Comment:
Do we not also want this when the creation flag is set?

Reply:
When the creation flag is set, the object will be added to a container and reindexed because of that: https://github.com/zopefoundation/Products.CMFCore/blob/master/Products/CMFCore/CMFCatalogAware.py#L272
Thus I would like to avoid indexing twice, though collective.indexing would solve that.

Reply:
Okay!